app/javascript/packs/application.tsx

```diff
@@ -17,6 +17,7 @@ import 'components/concepts-map/ConceptsMap.css'
 
 import React from 'react'
 import { initReact } from './react-bootloader.jsx'
+import { Editor } from '../components/common/Editor'
```

Comment:
I think this should probably be in `student`, not `common`, or not namespaced at all.

Reply:
I'll namespace it to student for now.